src/main/java/net/sf/jabref/importer/ImportFileFilter.java

```diff
@@ -33,7 +33,16 @@
 
     public ImportFileFilter(ImportFormat format) {
         this.format = format;
-        this.name = format.getFormatName();
+
+        StringBuilder sb = new StringBuilder(format.getFormatName());
+        if (format.getExtensions() != null) {
```

Comment:
No need to test for `!= null` since after #1487 this case is excluded by a test case. Better test for emptyness.